eth_utils/conversions.py

```diff
@@ -19,33 +26,37 @@
 )
 
 
+HexStr = NewType('HexStr', str)
```

Comment:
I think this should go somewhere else.  Maybe take a look at how `py-evm` sets up the `eth_typing` module?

Reply:
will update in future pr